sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -505,6 +505,27 @@ pi_result piQueueCreate(pi_context context, pi_device device,
   return cast<pi_result>(ret_err);
 }
 
+pi_result piQueueGetInfo(pi_queue queue, pi_queue_info param_name,
+                         size_t param_value_size, void *param_value,
+                         size_t *param_value_size_ret) {
+  if (queue == nullptr) {
+    return PI_ERROR_INVALID_QUEUE;
+  }
+
+  switch (param_name) {
+  case PI_EXT_ONEAPI_QUEUE_INFO_STATUS:
+    return PI_ERROR_INVALID_VALUE;
```

Comment:
Please add a comment why this is an error. Is it coming to OpenCL btw?

Reply:
As far as I know, we are going to ask to add extension to opencl. If it is not added then we may consider implementing a solution in OpenCL plugin itself.